README.md

```diff
@@ -114,6 +114,45 @@ after 'deploy:starting', 'sidekiq_systemd:quiet'
 after 'deploy:updated', 'sidekiq_systemd:stop'
 ```
 
+
+### Verify Ruby Versions
+
+There are three tasks available to check and verify the ruby version(s) installed and being used on the remote server:
+
+#### versions
+
+```
+bundle exec cap env ruby:versions
+=> host-1.stanford.edu,3.1.1,2.7.1,2.7.2,3.0.0,3.0.3,3.1.0,3.1.1,3.1.2
+=> host-2.stanford.edu,3.1.1,2.7.1,2.7.2,3.0.0,3.0.3,3.1.0,3.1.1,3.1.2
+```
+
+This comma seperated list is hostname,default ruby,installed versions. This list can then be uesd in automated deployment tools to report across hosts where version mismatches may occur.
+
+#### check_version
+```
+bundle exec cap env ruby:check_version
+=> host-1.stanford.edu - App: 3.1.1, Default: 3.1.1, Installed: 2.7.1, 2.7.2, 3.0.0, 3.0.3, 3.1.0, 3.1.1, 3.1.2
```

Comment:
I was thinking this would provide a human readable output and the other would provide machine readable for use in sdr-deploy something like we do for check_cocina. Ideally, I think that would be the same task name with a flag, but I'm not sure how/if we want to introduce that here. Totally open for suggestion/discussion.

Reply:
I was figuring sdr-deploy would just use verify-cocina, and if it doesn't pass, it could provide the human readable output and the gemfile versions?

Reply:
That may be a good approach as well. I'll work through it. :+1: